lib/ddtrace/contrib/concurrent_ruby/context_composite_executor_service.rb

```diff
@@ -0,0 +1,34 @@
+require 'concurrent/executor/executor_service'
+
+module Datadog
+  module Contrib
+    module ConcurrentRuby
+      # wraps existing executor to carry over trace context
+      class ContextCompositeExecutorService
+        extend Forwardable
+        include Concurrent::ExecutorService
+
+        attr_accessor :composited_executor
+
+        def initialize(composited_executor)
+          @composited_executor = composited_executor
+        end
+
+        def post(*args, &task)
```

Comment:
Can you explain a little about what this function does? Its purpose? Presumably in the context of Concurrent Ruby?